src/locales/ar.po

```diff
@@ -720,6 +733,11 @@ msgstr "رجاء أدخل عنوان بريدك الإلكتروني."
 msgid "Please log in."
 msgstr "الرجاء تسجيل الدخول."
 
+#, fuzzy
+#| msgid "Please note that your settings have been updated as follows:"
+msgid "Please note that client %1 settings has been updated as below:"
+msgstr "يرجى ملاحظة أن الإعدادات الخاصة بك تم تحديثها كما يلي:"
```

Comment:
Yes - if we could add a note to the guidelines explaining how to review fuzzy text that might help (every entry with fuzzy text should already be flagged as "needs review" in weblate, I think?), but I think for the other languages it hasn't been too much of a problem. Just need to be careful where the meaning has changed, like the "$value $1" above - the text we have now implies "current value", whereas it looks like the actual meaning is more like "old/previous value".